xml/System.Diagnostics/DiagnosticSource.xml

```diff
@@ -69,9 +69,9 @@
         <Parameter Name="name" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="name">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="name">The name of the event being written.</param>
+        <summary>Optional: if there is expensive setup for the notification, you can call IsEnabled before doing this setup. Consumers should not be assuming that they only get notifications for which IsEnabled is true however, it is optional for producers to call this API. The name should be the same as what is passed to <see cref="M:System.Diagnostics.DiagnosticSource.Write(System.String,System.Object)" />.</summary>
+        <returns><see langword="true" /> if it's enabled, <see langword="false" /> otherwise.</returns>
```

Comment:
@rpetrusha, I think "it's" refers to the notification event that we are checking. So my suggestion to clarify "it's" would be (if @wtgodbe approves):

```suggestion
        <returns><see langword="true" /> if the notification event is enabled, <see langword="false" /> otherwise.</returns>
```